__tests__/src/integration.spec.js

```diff
@@ -15,14 +15,37 @@
 const fs = require('fs');
 const path = require('path');
 const toMatchImageSnapshot = require('../../src').toMatchImageSnapshot;
+const rimraf = require('rimraf');
 
 expect.extend({ toMatchImageSnapshot });
 
 describe('integration tests', () => {
   const imagePath = path.resolve(__dirname, '../stubs', 'TestImage.png');
-  const data = fs.readFileSync(imagePath);
+  const imageData = fs.readFileSync(imagePath);
+
+  beforeAll(() => {
+    rimraf.sync('./__tests__/src/__image_snapshots__');
```

Comment:
why not specifically clean up the images created in the tests themselves? That way if we add more image snapshots later that we actually want to persist they dont get removed as a side effect of these tests.